lib/ts/utils.ts

```diff
@@ -168,7 +169,7 @@ function getPackageJsonString(input: {
             "start:backend": "cd backend && ${backendStartScript}",
             "start:backend-live-demo-app": "cd backend && ./startLiveDemoApp.sh",
             "start": "npm-run-all --parallel start:frontend start:backend",
-            "start-live-demo-app": "npx npm-run-all --parallel start:frontend-live-demo-app start:backend-live-demo-app"
+            "start-live-demo-app": "npm-run-all --parallel start:frontend-live-demo-app start:backend-live-demo-app"
```

Comment:
this command is for our live demo apps hosted by us, so please undo this change

Reply:
I'll gladly revert this change, but these commands are the same. when you run a package.json script with `npm run ...` npm will automatically use `npx` with that command. Writing `npx` here at the start is not necessary.

Reply:
oh yea, but i don't think the server in which we run the command installs `npm-run-all` when `npm i` is ran. So it will fail without `npx` in front. I don't recall entirely, so prefer to just keep this the way it is

Reply:
Done!